Source/FieldSolver/ElectrostaticSolver_EB.cpp

```diff
@@ -0,0 +1,658 @@
+/* Copyright 2021 Modern Electron
```

Comment:
Can we move this file to ElectrostaticSolver_ME ? Then it's consistent with the compile var, and we won't get confused when EB is introduced upstream.